systemtest/aggregation_test.go

```diff
@@ -151,13 +151,17 @@ func TestTransactionAggregationShutdown(t *testing.T) {
 }
 
 func TestServiceDestinationAggregation(t *testing.T) {
+	t.Setenv("ELASTIC_APM_GLOBAL_LABELS", "department_name=apm,organization=observability,company=elastic")
```

Comment:
Can we add a numeric label so we can verify that both mappings are applied?

Reply:
Do you have an easy way to add numeric labels with the Go agent? I'm looking at Go agent tracer code and in `parseGlobalLabels` it parses labels from env into kv of string and string.

Reply:
Yeah, you're right. Looking at the implementation we only ever added support for k/v string label parsing. It'd be ideal if we could test that both are parsed in APM Server, but that'd require sending a literal payload and creating another test. I think it's worth the trouble. What do you think?

Reply:
If we do it I would love to do it for all aggregations, because this test is missing for all of them.

Reply:
let's open a follow up issue then to add numeric labels tests to all aggregations.

Reply:
Created #10324